src/pybind/mgr/dashboard/frontend/src/app/shared/services/task-manager-message.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@angular/core';
+import { FinishedTask } from '../models/finished-task';
+import { Task } from '../models/task';
+
+class TaskManagerMessage {
+  descr: (metadata) => string;
+  success: (metadata) => string;
+  error: (metadata) => object;
+
+  constructor(descr: (metadata) => string,
+              success: (metadata) => string,
+              error: (metadata) => object) {
+    this.descr = descr;
+    this.success = success;
+    this.error = error;
+  }
+}
+
+@Injectable()
+export class TaskManagerMessageService {
+
+  messages = {
+  };
+
+  constructor() { }
+
+  getSuccessMessage(finishedTask: FinishedTask) {
```

Comment:
I think we should check if there is a value for the `finishedTask.name` key.
Otherwise if some async task method is implemented in the backend this will result in exceptions on the frontend.